keps/sig-multicluster/5313-placement-decision-api/README.md

```diff
@@ -0,0 +1,429 @@
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+Today every multicluster scheduler publishes its own API to convey where a workload should run,
+forcing downstream tools such as GitOps engine, workload orchestrator, progressive rollout controller,
+or AI/ML pipeline having to understand a scheduler specific API.
+
+This KEP introduces a vendor neutral `PlacementDecision` API that standardizes
+the output of multicluster placement calculations.  
+A `PlacementDecision` object is data only: a namespaced list of chosen clusters
+whose `clusterName` values must map one to one to `ClusterProfile` names defined by the
+[ClusterProfile API](https://github.com/kubernetes/enhancements/pull/4322).
+Any scheduler can emit the object and any consumer can watch it.
+Neither side needs to know about the other, enabling true plug and play in the multicluster stack.
+
+![PlacementDecision Flow Diagram](./placementdecision.jpg)
+
+
+## Motivation
+
+A typical multicluster setup:
+
+1. Scheduler: examines the fleet (`ClusterProfile` objects) and decides *where* a workload should land.
+2. Consumer: GitOps engine, workload orchestrator, progressive rollout controller, AI/ML pipeline
+   read that decision and act (usually by creating [Work](https://github.com/kubernetes-sigs/work-api) objects).
+
+Currently every scheduler have its own API for step #1
+so each consumer might need to learn different APIs which slow down integration,
+locking users to specific vendors, and complicate RBAC/validation work.
+
+A standardize unversial `PlacementDecision` API:
+
+* Decouples schedulers from consumers swap either side without rewriting the other.
+* Aligns with the SIG-Multicluster `ClusterProfile` inventory:
+  the decision's cluster keys are guaranteed to exist.
+* Simplifies RBAC: one resource and label schema to secure.
+* Enables GPU aware AI scheduling, cost based placement, and rapid failover.
```

Comment:
I don't think that's really true.
There is a fundamental problem with scheduling for HW, which is that the cluster's scheduler has the final say. The hub's placement scheduler is likely to have an incomplete view.

Also, PlacementDecision doesn't do anything about GPU/HW etc. That would be up to scheduler implementations :)

Reply:
Removed as suggested.